src/modules/app/containers/RouteContentSwitcher.js

```diff
@@ -27,12 +29,24 @@ class RouteContentSwitcher extends React.PureComponent<PropsType> {
     return null
   }
 
+  renderExtraFailure = (payloads: {[string]: Payload<any>}): React.Node => {
+    if (payloads.extras && !payloads.extras.isFetching && payloads.extras.data && !payloads.extras.error) {
+      if (payloads.offers && !find(payloads.extras.data, extra => extra.alias === WOHNEN_EXTRA)) {
+        return <FailureSwitcher error={new Error('The Wohnen extra is not supported.')} />
```

Comment:
We should use a `ContentNotFoundError` here. The `FailureSwitcher` should then render a localized message.